alg/gdal_interpolateatpoint.cpp

```diff
@@ -136,6 +136,28 @@ GDALInterpExtractDEMWindow(GDALRasterBand *pBand,
     return true;
 }
 
+static double CubicKernel(double dfX)
```

Comment:
Should this function be in the header file? Currently is not used anywhere else. It is copied from `alg/gdalwarpkernel.cpp`

Reply:
> Should this function be in the header file

we should try to limit copy&paste. Perhaps as a inline function in a alg/gdalresamplingkernels.h header used by gdal_interpolateatpoint.cpp and gdalwarpkernel.cpp (I also considered in the existing alg/gdal_alg_priv.h). Or any equivalent suggestion you may have

Reply:
done